hoot-core/src/main/cpp/hoot/core/io/OgrUtilities.h

```diff
@@ -68,7 +71,7 @@ class OgrUtilities
 public:
   OgrUtilities();
 
-  shared_ptr<GDALDataset> createDataSource(const QString& url);
+  shared_ptr<GDALDataset> createDataSource(const QString& url, bool read);
```

Comment:
The new parameter (read) deserves a comment, like "If you only plan to read from this data source, set read=true for better performance, read=false if you need to read and/or write"